README.md

```diff
@@ -31,84 +31,84 @@ userPassword: readonly
 EOF
 ```
 
-Afterwards reload kidmd by running `systemctl reload kopano-kidmd`.
+Afterwards reload idmd by running `systemctl reload libregraph-idmd`.
 
 The `config.ldif` is for service users only and the data in there is used for bind requests only, but never returned for search requests.
 
 #### Main configuration
 
-The main directory data comes from the `main.d` folder. All `.ldif` files in there are loaded in lexical order and parsed as templates to create the data provided by the kidmd LDAP endpoint.
+The main directory data comes from the `main.d` folder. All `.ldif` files in there are loaded in lexical order and parsed as templates to create the data provided by the idmd LDAP endpoint.
 
-Whenever any of the ldif files are changed, added or removed, make sure to reload kidmd with `systemctl reload kopano-kidmd`.
+Whenever any of the ldif files are changed, added or removed, make sure to reload idmd with `systemctl reload libregraph-idmd`.
 
-Kidmd listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration is installed (commented) and also available in the `/usr/share/doc/kopano-kidmd/examples` directory.
+idmd listens on `127.0.0.1:10389` by default and does not ship with any default users. Example configuration is installed (commented) and also available in the `/usr/share/doc/libregraph-idmd/examples` directory.
 
 ##### Add new users using the `gen newusers` command
 
-Kidmd provides a way to create ldif data for new users using batch mode similar to the unix `newusers` command using the following standard password file format:
+idmd provides a way to create ldif data for new users using batch mode similar to the unix `newusers` command using the following standard password file format:
 
 ```bash
-uid:userPassword:uidNumber:gidNumber:cn,[mail][,kopanoAliases...]:ignored:ignored
+uid:userPassword:uidNumber:gidNumber:cn,[mail][,libregraphAliases...]:ignored:ignored
```

Comment:
libregraphAliases does not exist - i used `mailAlternateAddress` in the examples/code.